pom.xml

```diff
@@ -122,7 +122,7 @@
         <testcontainers.version>1.16.0</testcontainers.version>
         <jetbrains.annotations.version>20.1.0</jetbrains.annotations.version>
         <antlr.version>4.9.2</antlr.version>
-        <nlp.version>3.9.2</nlp.version>
+        <nlp.version>4.3.1</nlp.version>
```

Comment:
was there any specific reason to bump up this dependency?

Reply:
Some warning issued due to deprected/new release code.